hw2/02-reverse.js

```diff
@@ -1,3 +1,26 @@
 /** Exercise 02 - Reverse **/
 
-// Add your code here
+function reverseNum()
+{
+  let number = document.getElementById('input').value;
```

Comment:
Another nicely-formatted file! Very easy to read and in most instances produces the correct output. It catches instances of not putting in any numbers, too few numbers, and too many numbers. However, I did notice that leading zeros get removed from the result.

For example. If I enter `00001111` the result is:

```
00001111 --> 1111
```
Besides that this looks great though!